src/js/components/molecules/social-nav/SocialNav.vue

```diff
@@ -0,0 +1,42 @@
+<template>
```

Comment:
Voeg attribuut 'functional' toe, dan is een goede optimalisatie voor die 'domme' component